openvpn-install.sh

```diff
@@ -1309,9 +1338,12 @@ function manageMenu() {
 		revokeClient
 		;;
 	3)
-		removeOpenVPN
+			listcerts
```

Comment:
Please use the same casing as the other functions and remove the added space

Reply:
Changed Casing